SocialCareCaseViewerApi/V1/Domain/Visit.cs

```diff
@@ -1,14 +1,37 @@
-using System;
+#nullable enable
 
 namespace SocialCareCaseViewerApi.V1.Domain
 {
     public class Visit
     {
-        public string Id { get; set; }
-        public DateTime CreatedOn { get; set; }
-        public string CreatedByEmail { get; set; }
-        public string MosaicId { get; set; }
-        public string Title { get; set; }
-        public string Content { get; set; }
+        public long VisitId { get; set; }
```

Comment:
I take it this has been validated with FE devs so it has all the properties they need?

Reply:
I've shown Luca the data being returned. 

For the /cases endpoint I will still get the Visit class to transform into what the FE needs (for when showing a list of records). 

For getting a specific Visit the frontend should want all the Visit information so plan is to return it all in the format from the new Visit class. I will confirm.